wallet/src/repository/keys.rs

```diff
@@ -132,6 +132,12 @@ pub fn wallet_last_sync() -> Key<&'static str, CheckpointBeacon> {
     Key::new("last-sync")
 }
 
+/// The epoch of the latest consolidated block that was processed by the wallet
```

Comment:
Isn't this always equal to `wallet_last_sync`? I thought the wallet only stores consolidated state to the database?

Reply:
I think so. Apparently, this is redundant.